.github/workflows/docker-publish.yml

```diff
@@ -94,4 +94,36 @@ jobs:
           target: kubectl-argo-rollouts
           platforms: ${{ steps.platform-matrix.outputs.platform-matrix }}
           push: ${{ github.event_name != 'pull_request' }}
-          tags: ${{ steps.plugin-meta.outputs.tags }}
\ No newline at end of file
+          tags: ${{ steps.plugin-meta.outputs.tags }}
+
+      - name: Install cosign
+        uses: sigstore/cosign-installer@main
+        with:
+          cosign-release: 'v1.13.0'
+
+      - name: Sign Argo Rollouts latest controller-image
+        run: |
+          cosign sign --key env://COSIGN_PRIVATE_KEY ${TAGS}
```

Comment:
This output provides two separate tags(master and latest) , we all need to sign by digest instead.
please do not merge till changes are made.